### PR TITLE
Indirect - MSDFit - Display and update properties in property table

### LINKED ISSUE #20642: MSDFit - Correct parameters not displayed and Plot Spectrum not functioning
The parameters for the three models (Gaussian, Peters & Yi) are not being displayed in the Indirect > Data Analysis > MSDFit interface. These parameters should be available for modification in the property table and should be updated upon a fit.

The plot spectrum spinner is not properly selecting spectrum after a fit.

### PR DESCRIPTION
**Pull request by brandonhewer**
In the MSDFit Indirect Data Analysis interface, when the model is updated, the property table does not display the properties for the current model (e.g. Intensity and MSD for a Gaussian model).
These properties should be displayed and updated and the plot preview should properly display the results of a fit.

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > MSDFit.
2. Select any model.
3. In the input field enter the supplied reduced file (see below).
4. Ensure the spectra range covers the entire input.
5. Click 'Run'.
6. Scroll through all spectrum, using the 'Plot Spectrum' spinner and check that the sample and fit are plotted for each and the values in the property table are updated and correspond to the created \*\_Parameters workspace.
7. Repeat steps 5 and 6 but with a sub-range for the spectra (doesn't cover entire range), also check that spectrum outside the fitted range plot the sample but not a fit.
8. Repeat step 6, but fit a single spectrum using 'Fit Single Spectrum' instead of 'Run'.

**Test File**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/1341666/irs26176_graphite002_red.zip)

Fixes #20642. 

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: qt/scientific_interfaces/Indirect/ConvFit.cpp [hunk 1]
```diff
@@ -283,9 +284,9 @@ void ConvFit::initFABADAOptions() {
 }
 
 /**
-* Handles the initial set up and running of the ConvolutionFitSequential
-* algorithm
-*/
+ * Handles the initial set up and running of the ConvolutionFitSequential
+ * algorithm
+ */
```
Comment (LamarMoore) [inline qt/scientific_interfaces/Indirect/ConvFit.cpp]: Please undo clang formatting of documentation where relevant.

File: qt/scientific_interfaces/Indirect/ConvFit.cpp [hunk 2]
```diff
@@ -526,9 +507,9 @@ void ConvFit::addSampleLogsToWorkspace(const std::string &workspaceName,
 }
 
 /**
-* Validates the user's inputs in the ConvFit tab.
-* @return If the validation was successful
-*/
+ * Validates the user's inputs in the ConvFit tab.
+ * @return If the validation was successful
+ */
```
Comment (LamarMoore) [inline qt/scientific_interfaces/Indirect/ConvFit.cpp]: revert doc changes

File: qt/scientific_interfaces/Indirect/ConvFit.cpp [hunk 3]
```diff
@@ -565,28 +546,27 @@ bool ConvFit::validate() {
 }
 
 /**
-* Reads in settings files
-* @param settings The name of the QSettings object to retrieve data from
-*/
+ * Reads in settings files
```
Comment (LamarMoore) [inline qt/scientific_interfaces/Indirect/ConvFit.cpp]: And again

**Review by LamarMoore — CHANGES_REQUESTED** [review]
Functionality as described in the PR test instructions. However, I noticed there are many irrelevant clang -format changes to documentation which should be reverted. It makes the diffs huge and difficult to read. Please revert the documentation changes.

**Comment by brandonhewer** [comment]
@LamarMoore You're right about the Doc Changes (this is likely due to formatting in Visual Studio). Currently there are number of important bug fixes to be made within Indirect by Wednesday. Martyn has let me know he would be willing to Merge this PR if everyone works as expected. Would it be possible to avoid reverting these changes this time and use the `?w=1` suffix when viewing the changes?

**Review by LamarMoore — APPROVED** [review]

**Pull request merged**